helm/ph-ee-kenya/values.yaml

```diff
@@ -0,0 +1,525 @@
+    SPRING_PROFILES_ACTIVE: "bb,tenants"
+    tenantPrimary:
+      clientId: "mifos"
+      clientSecret: "password"
+      tenant: "gorilla"
+    server:
+      ssl:
+        keyPassword: "password"
+        keyStorePassword: "password"  
+    ingress:
+      enabled: false
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: channel.sandbox.mifos.io          
+          paths: 
+          - path: "/"
+            backend:
+              service:
+                name: "ph-ee-connector-channel"
+                port:
+                  number: 8443
+        - host: channel-gsma.sandbox.mifos.io          
+          paths:          
+          - path: "/"
+            backend:
+              service:
+                name: "ph-ee-connector-channel-gsma"
+                port:
+                  number: 82
+    deployment:
+      annotations:
+        deployTime: "{{ .Values.deployTime }}"
+
+
+  operations_app:
+    enabled: true
+    image: docker.io/openmf/ph-ee-operations-app:latest
+    hostname: "ops-bk.sandbox.mifos.io"
+    datasource:
+      username: "mifos"
+      password: "password"
+      host: "operationsmysql"
+      port: 3306
+      schema: "tenants"
+    ingress:
+      enabled: false
+      annotations: 
+        kubernetes.io/ingress.class: nginx 
+        nginx.ingress.kubernetes.io/cors-allow-headers: 'platform-tenantid, x-correlation-id'
+        nginx.ingress.kubernetes.io/cors-allow-methods: PUT, GET, POST, OPTIONS, DELETE
+        nginx.ingress.kubernetes.io/cors-allow-origin: '*'
+        nginx.ingress.kubernetes.io/enable-cors: "true"
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: ops-bk.sandbox.mifos.io          
+          paths: 
+          - path: "/"
+            backend:
+              service:
+                name: "ph-ee-operations-app"
+                port:
+                  number: 80
+    deployment:
+      annotations:
+        deployTime: "{{ .Values.deployTime }}"
+
+  operations_web:
+    enabled: true
+    image: docker.io/openmf/ph-ee-operations-web:latest
+    hostname: "ops.sandbox.mifos.io"
+    ingress:
+      enabled: false
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: ops.sandbox.mifos.io          
+          paths: 
+          - path: "/"
+            backend:
+              service:
+                name: "ph-ee-operations-web"
+                port:
+                  number: 4200
+
+  identity:
+    hostname: "ops-bk.sandbox.mifos.io"
+
+  ph_ee_connector_gsma:
+    enabled: true
+    image: docker.io/openmf/ph-ee-connector-gsma:latest
+    SPRING_PROFILES_ACTIVE: "bb"
+    limits:
+      cpu: "500m"
+      memory: "512M"
+    requests:
+      cpu: "100m"
+      memory: "256M"
+    deployment:
+      annotations:
+        deployTime: "{{ .Values.deployTime }}"
+
+  ph_ee_connector_slcb:
+    enabled: false
+   
+  mpesa:
+    enabled: true
+
+  roster_connector:
+    enabled: false
```

Comment:
if mpesa is enabled should not one of the roster or paygops be enabled?

Reply:
Yeah but it was specified in the project plan to remove paygops and roaster ([here](https://docs.google.com/document/d/1evZE-BJ9haun7Brps3d0HzGXdgGmb5CN/edit)). I think I need to ask avik regarding this.